### DISCUSSION TITLE
How to import subflows

### DISCUSSION THREAD

**Discussion by shenhai-ran** [AgentEra/Agently#159] [Q&A] [answered]
Hello,

If I have some subflows defined in different python files, such as 
``` python
# some codes
(
    sub_workflow
        .connect_to("actions")
        .connect_to("END")
)
```

and I have a main workflow in another python file and I would like to connect all the subflows in this python file.
How to include those subflows into the main flow python file?

THanks

**Comment by Maplemx** [comment, marked as answer]
Suppose you have two files in the same dir:

/path/to/your/dir/sub_workflow.py:

```py
import Agently
sub_workflow = Agently.Workflow()

...

```

/path/to/your/dir/main_workflow.py:

```py
import Agently
# you can import sub_workflow by this way
from sub_workflow import sub_workflow

main_workflow = Agently.Workflow()
```
